iron-form.html

```diff
@@ -137,13 +147,18 @@
 
       this._requestBot.generateRequest();
       this.fire('iron-form-submit', json);
+
+      if (this.allowActionRedirect) {
+        var redirectUrl = this.action + '?' + this._requestBot.queryString;
+        window.location = redirectUrl;
```

Comment:
Ugh, turns out changing `window.location` makes an extra `get` request so this solution still blows.